lib/galaxy/schema/schema.py

```diff
@@ -3657,3 +3657,34 @@ class DatasetSummary(Model):
     file_size: int
     total_size: int
     uuid: UuidField
+
+
+class UserReportingErrorPayload(Model):
+    dataset_id: DecodedDatabaseIdField = Field(
+        default=...,
+        title="History Dataset Association ID",
+        description="The History Dataset Association ID related to the error.",
+    )
+    message: Optional[str] = Field(
+        default=None,
+        title="Message",
+        description="The optional message sent with the error report.",
+    )
+    email: Optional[str] = Field(
+        default=None,
+        title="Email",
+        description="Email address for communication with the user. Only required for anonymous users.",
+    )
+    transcript: Optional[str] = Field(
+        default=None,
+        title="Transcript",
+        description="The optional tool transcript sent with the error report.",
```

Comment:
I don't know what you mean by transcript here - is this language you have gotten from somewhere or are you adding it.

Reply:
_It's just the existing wording on usegalaxy.org:_

![screen](https://github.com/galaxyproject/galaxy/assets/3672779/d3dffade-3946-4215-8d14-5385fa5bbb8f)

Reply:
_It's just the existing wording on usegalaxy.org:_

![Greenshot 2024-04-09 17 02 17](https://github.com/galaxyproject/galaxy/assets/3672779/e962fcfb-9b7e-4e0b-a62f-0392574dfb8b)